inventory/components/header.tsx

```
@@ -0,0 +1,223 @@
+                    }}
+                  >
+                    {darkMode ? (
+                      <Moon size={16} color="#3887fa" />
+                    ) : (
+                      <Sun size={16} color="#f6c026" />
+                    )}
+                  </div>
+                </div>
+              </div>
+
+              <button className={styles['blue-btn']}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)